gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 awsSdk2 = { strictly = '2.20.103' }
 junit = { strictly = '5.9.3' }
 lambda = { strictly = '1.2.2' }
-assertj = { strictly = '3.23.1' }
+assertj = { strictly = '3.24.2' }
```

Comment:
newer version has `usingRecursiveAssertion` in addition to `usginRecursiveComparison`